packages/react/context-menu/src/ContextMenu.tsx

```diff
@@ -82,6 +82,17 @@ const ContextMenuTrigger = React.forwardRef((props, forwardedRef) => {
   const virtualRef = React.useRef({
     getBoundingClientRect: () => DOMRect.fromRect({ width: 0, height: 0, ...pointRef.current }),
   });
+  const longPressTimerRef = React.useRef(0);
+  const clearLongPress = React.useCallback(
+    () => window.clearTimeout(longPressTimerRef.current),
+    []
+  );
+  const open = (event: React.MouseEvent | React.PointerEvent) => {
```

Comment:
Can we rename this to `handleOpen` for consistency with our other event handlers please?